forms-flow-web/src/components/Draft/Edit.js

```diff
@@ -244,7 +244,7 @@ const View = React.memo((props) => {
             onConfirm={props.onConfirm}
           ></SubmissionError>
           {isAuthenticated ? (
-            <Link title={t("Back to Drafts")} to={`${redirectUrl}draft`}>
+            <Link title={t("Back to Drafts")} to={`${redirectUrl}draft`} className="mt-1">
```

Comment:
is this right approach because already have margin top in main containers. so if you need to align the left arrow and draft title make them inside a div and give d-flex class

Reply:
yeah sure ,will do